tasks/section5.yml

```diff
@@ -300,8 +312,60 @@
       - patch
       - rule_5.2.14
 
-- name: "SCORED | 5.2.15 | PATCH | Ensure SSH access is limited"
-  command: /bin/true
+- name: "SCORED | 5.2.15 | PATCH | Ensure SSH access is limited - allowusers"
+  lineinfile:
+      state: present
+      dest: /etc/ssh/sshd_config
+      regexp: "^AllowUsers"
+      line: AllowUsers {{ rhel7cis_sshd['allowusers'] }}
+  notify:
+      - restart sshd
+  when: "rhel7cis_sshd['allowusers']|default('') != ''"
+  tags:
+      - level1
+      - level2
+      - patch
+      - rule_5.2.15
+
+- name: "SCORED | 5.2.15 | PATCH | Ensure SSH access is limited - allowgroups"
+  lineinfile:
+      state: present
+      dest: /etc/ssh/sshd_config
+      regexp: "^AllowGroups"
+      line: AllowGroups {{ rhel7cis_sshd['allowgroups'] }}
+  notify:
+      - restart sshd
+  when: when: "rhel7cis_sshd['allowgroups']|default('') != ''"
```

Comment:
Just make the correction and commit to your fork/branch and the update will show up here!